LoopCore/NSUserDefaults.swift

```diff
@@ -20,6 +20,8 @@ extension UserDefaults {
         case loopSettings = "com.loopkit.Loop.loopSettings"
         case insulinSensitivitySchedule = "com.loudnate.Naterade.InsulinSensitivitySchedule"
         case overrideHistory = "com.tidepool.loopkit.overrideHistory"
```

Comment:
🤷 I don't know. :) Let me know if you think it's important to change; it will mean a loss of overrides history for any users if we change it, and don't create a migration.

Reply:
Leave it as-is, I guess.